improver/calibration/ensemble_calibration.py

```diff
@@ -1096,3 +1106,161 @@ def process(self, current_forecast, coefficients_cube, landsea_mask=None):
                 flip_mask, location_parameter_cube.data)
 
         return location_parameter_cube, scale_parameter_cube
+
+
+class ApplyEMOS(BasePlugin):  # TODO change to PostProcessingPlugin later
+    """
+    Class to calibrate an input forecast given EMOS coefficients
+    """
+
+    @staticmethod
+    def _get_forecast_type(forecast):
```

Comment:
Added.  Question: under what circumstances would `ignore_ecc_bounds` ever need to be `False` for EMOS?

Reply:
In practice, we usually set `ignore_ecc_bounds` to be `True` to avoid unexpected failures, but I think it really depends upon how concerned the user is by values within the forecast exceeding the [ECC bounds](https://github.com/metoppv/improver/blob/master/improver/ensemble_copula_coupling/constants.py#L51). Therefore for operational running, I would expect `ignore_ecc_bounds` to always be set to `True` but when experimenting someone might want to set `ignore_ecc_bounds` to False.